### PR TITLE
[MRG] Frontend: teaching page

### PR DESCRIPTION
**Pull request by lucyleeow**
Includes information from the old about page and the teaching page.

image

Old page:

image

cc @agramfort @kegl

### CODE REVIEW

File: ramp-frontend/ramp_frontend/templates/teaching.html [hunk 1]
```diff
@@ -27,27 +27,107 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
...
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Using RAMP for teaching</h1>
+                If you are interested in using the platform for teaching,
+                We can open a new event for any <a href="/problems">
+                RAMP challenge</a>. You will have full control over the users
+                who can sign up for your event and the timeline the event
+                (start, moving from closed to open phase and end).
```
Comment (agramfort) [inline ramp-frontend/ramp_frontend/templates/teaching.html]: this assumes that teachers are admins. You cannot be admin of only one event for now.

File: ramp-frontend/ramp_frontend/templates/teaching.html [hunk 2]
```diff
@@ -27,27 +27,107 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
...
+            </p>
+
+            <p>
+                At the end of your challenge we will give you a csv file with all
+                the performance metrics of each student.
```
Comment (agramfort) [inline ramp-frontend/ramp_frontend/templates/teaching.html]: it should be able to download this from the admin page from the event. It's very manual so far

File: ramp-frontend/ramp_frontend/templates/teaching.html [hunk 3]
```diff
@@ -27,27 +27,107 @@ <h4 class="modal-title" id="myModalLabel">{{ category }}</h4>
...
+                thousands of models that are already in the platform.
+            </p>
+
+            <br>
+
+            <p>
+                <h1 style='font-weight: bold;'>Contact us</h1>
+
+                To use RAMP for teaching or to work with us to develop a new
+                RAMP challenge, contact us at <a href="mailto:admin@ramp.studio"><strong>admin@ramp.studio</strong></a>.
```
Comment (agramfort) [inline ramp-frontend/ramp_frontend/templates/teaching.html]: I would also that RAMP code is open source so you can easily set up your own server. That's the easiest way to make your own challenge. It does not require our time

File: ramp-frontend/ramp_frontend/templates/teaching.html (hunk 1 above)
Comment (lucyleeow) [reply ramp-frontend/ramp_frontend/templates/teaching.html]: So just remove the sign up part?

File: ramp-frontend/ramp_frontend/templates/teaching.html (hunk 2 above)
Comment (lucyleeow) [reply ramp-frontend/ramp_frontend/templates/teaching.html]: I don't see this option. Should it be under 'dashboard admin' or 'update event'? @glemaitre @maikia do you know if this option exists?

File: ramp-frontend/ramp_frontend/templates/teaching.html (hunk 1 above)
Comment (agramfort) [reply ramp-frontend/ramp_frontend/templates/teaching.html]: yes. Don't say that they will be able to approve their users

File: ramp-frontend/ramp_frontend/templates/teaching.html (hunk 2 above)
Comment (agramfort) [reply ramp-frontend/ramp_frontend/templates/teaching.html]: yes it is not available yet but if we say this it should be added

**Comment by lucyleeow** [comment]
image

**Comment by lucyleeow** [comment]
Good to merge @agramfort!

**Comment by agramfort** [comment]
thx @lucyleeow

**Pull request merged**